docs-web/installation/building-ingress-controller-image.md

```diff
@@ -26,6 +26,7 @@ We build the image using the make utility and the provided `Makefile`. Let’s c
 1. Clone the Ingress Controller repo:
     ```
     $ git clone https://github.com/nginxinc/kubernetes-ingress/
+    $ cd kubernetes-ingress
```

Comment:
can't we just do ```git clone https://github.com/nginxinc/kubernetes-ingress/ --branch v1.11.3``` in one line?

Reply:
I don't see why not. We do that in a couple of places though, not only here. Would you like to create a PR?
cd is still needed